wiki/Help_centre/Store/de.md

```diff
@@ -2,128 +2,54 @@
 tags:
   - store
   - product
-  - keyboard
-  - tablet
-  - area
-  - device
-  - setup
-  - pendrive
-  - Gerät
-  - Tastatur
-outdated_translation: true
-outdated_since: bcf19a2e1abb302c68a5e6f554cd4f1ec5336a19
+  - Produkt
+  - keycap
+  - keycaps
+  - Taste
+  - Tasten
+  - deskmat
+  - deskmats
+  - Tischmatte
+  - Unterlage
 ---
 
 # osu!store und Merchandise
 
 *Hauptseite: [Hilfecenter](/wiki/Help_centre)*\
-*Anmerkung: Diese Produkte werden nicht mehr länger zum Verkauf im osu!store angeboten.*
+*Für Hilfe zu älteren Produkten, siehe: [Shoparchiv](/wiki/Help_centre/Store/Store_archive)*
 
-Hast du Schwierigkeiten mit einem Produkt aus dem osu!store? Prüfe, ob es eine Lösung für dein Problem gibt!
+Dieser Artikel bezieht sich nur auf Produkte, die im [osu!store](https://osu.ppy.sh/store/listing) erworben werden können.
 
-## osu!keyboard {id=osu!keyboard}
+## Ich möchte meine Anschrift ändern! {id=address}
 
-### Wie konfiguriere ich das osu!keyboard? {id=osu!keyboard-setup}
+Sende eine E-Mail mit der Bestellnummer und der neuen Adresse an [merch@ppy.sh](mailto:merch@ppy.sh).
 
-**Du kannst das osu!keyboard-Konfigurationsprogramm verwenden, das auf [puush](https://puu.sh/l6urN/4b6bc800f2.zip) zur Verfügung steht.**
+Wenn deine Bestellung noch nicht versandt wurde, aktualisieren wir die Adresse und senden die Bestellung an die neue Anschrift. Wenn die Bestellung bereits unterwegs ist, können keine Änderungen mehr vorgenommen werden.
```

Comment:
Bit less formal maybe

```suggestion
Wenn deine Bestellung noch nicht versandt wurde, aktualisieren wir die Adresse und senden die Bestellung an die neue Anschrift. Wenn die Bestellung bereits unterwegs ist, können wir leider nicht mehr viel machen.
```

.. is `machen` correct there? maybe `tun`

Reply:
According to [this article](https://dein-sprachcoach.de/unterschied-zwischen-machen-und-tun/), `tun` is used for formal texts and `machen` in informal conversations. `Tun` is also used for emphasising specific actions.

You can say both, though my gut feeling tells me that `etwas für jemanden tun` is more common then `etwas für jemanden machen`.

Reply:
I was proposing the less formal variant on purpose after looking at the English text. Both are fine, but the current version seems a bit stiff.